man-roxygen/handler_fn.R

```diff
@@ -0,0 +1,32 @@
+# See notes on the use of `r ...` dynamic R code in roxygen2 comments
+# `utils_conditions.R`. We have to use a bit different code to get this to work
+# inside this template tag file, leading to long `r ...` chunks; don't
+# wrap/fill/format these, or else current roxygen2 at time of writing will not
+# properly process them. Keep the blank lines around them to help prevent
+# accidental auto-formatting.
```

Comment:
Roxygen templates provide a different way to execute dynamic code with <%= ...... %>. Might provide a way around the line wrap fighting.  (Here and in other template file, and maybe also in other non-template places where dynamic R code is used that could be turned into templates.)